express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+let grades = {};
+let nextId = 1;
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  nextId++;
+  grades = req.body;
```

Comment:
The new grade object should be stored at its id in the grades object. Posting multiple grades should retain them all until the server is restarted.

Reply:
![express-post-json](https://user-images.githubusercontent.com/82407007/123707892-4f3e4980-d81f-11eb-8158-b1cae931aba1.PNG)